payment/library/stripe.py

```diff
@@ -7,21 +7,15 @@ class Stripe:
     def __init__(self):
         stripe.api_key = settings.STRIPE_SECRET_KEY
 
-    def create_coupon(self, discount):
+    def create_coupon(self, discount_in_dollars):
         try:
-            if discount['type'] == 'percent':
-                stripe_coupon = stripe.Coupon.create(
-                    percent_off=discount['amount'],
-                    duration='forever'
-                )
-            elif discount['type'] == 'dollars':
-                stripe_coupon = stripe.Coupon.create(
-                    amount_off=discount['amount'] * 100,
-                    duration='forever',
-                    currency=settings.SINGAPORE_DOLLAR_CURRENCY
-                )
+            stripe_coupon = stripe.Coupon.create(
+                amount_off=discount_in_dollars * 100,
```

Comment:
This logic is confusing to me. Why multiply discount in dollars by 100?

Reply:
Stripe expects amounts in cents - i'll add a comment